build.sbt

```
@@ -93,9 +93,9 @@ lazy val `scalajs-cli`: Project = project.in(file(".")).
     ),
 
     // assembly options
-    mainClass in assembly := None, // don't want an executable JAR
-    assemblyOption in assembly ~= { _.copy(includeScala = false) },
-    assemblyJarName in assembly :=
+    assembly /  mainClass := None, // don't want an executable JAR
```

There's always one space too many on the right of `/`. For example here:
```suggestion
    assembly / mainClass := None, // don't want an executable JAR
```
Could you amend that?

Otherwise, LGTM.

---

done